content/en/docs/concepts/services-networking/service.md

```diff
@@ -217,184 +247,51 @@ In the example above, traffic is routed to the single endpoint defined in
 the YAML: `192.0.2.42:9376` (TCP).
 
 {{< note >}}
-The Kubernetes API server does not allow proxying to endpoints that are not mapped to 
-pods. Actions such as `kubectl proxy <service-name>` where the service has no 
-selector will fail due to this constraint. This prevents the Kubernetes API server 
-from being used as a proxy to endpoints the caller may not be authorized to access. 
+The Kubernetes API server does not allow proxying to endpoints that are not mapped to
+pods. Actions such as `kubectl proxy <service-name>` where the service has no
+selector will fail due to this constraint. This prevents the Kubernetes API server
+from being used as a proxy to endpoints the caller may not be authorized to access.
 {{< /note >}}
 
-An ExternalName Service is a special case of Service that does not have
+An `ExternalName` Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
-### Over Capacity Endpoints
+### Over-capacity endpoints
+
 If an Endpoints resource has more than 1000 endpoints then a Kubernetes v1.22 (or later)
 cluster annotates that Endpoints with `endpoints.kubernetes.io/over-capacity: truncated`.
 This annotation indicates that the affected Endpoints object is over capacity and that
 the endpoints controller has truncated the number of endpoints to 1000.
 
+### Endpoints
+
+In the Kubernetes API, an
+[Endpoints](/docs/reference/kubernetes-api/service-resources/endpoints-v1/)
+(the resource kind is plural) defines a list of network endpoints, typically
+referenced by a Service to define which Pods the traffic can be sent to.
+
 ### EndpointSlices
 
 {{< feature-state for_k8s_version="v1.21" state="stable" >}}
 
+(If you're just beginning to learn about Services, it's OK to skip learning
+about the differences between Endpoints and EndpointSlice; the overall effect
+is similar and the control plane synchronizes changes between the two APIs
```

Comment:
I'm not sure if we should add this paragraph here and just say that EndpointSlices is an evolution of Endpoints , to handle better scalability and other network features. Another note is that Endpoints API is in fact frozen and any new features are only developed in the EndpointSlices feature

cc: @robscott

Reply:
I wasn't sure which API to focus on. I can see the case for either.

Reply:
I would focus on EndpointSlices since all new features will rely on them. EndpointSlice functionality is a superset of Endpoints functionality.